.github/workflows/jacktrip.yml

```diff
@@ -235,27 +261,6 @@ jobs:
           echo "version=$VERSION" >> $GITHUB_OUTPUT
           echo "name=$NAME" >> $GITHUB_OUTPUT
           echo "stamp=$(date '+%Y-%m-%d')" >> $GITHUB_OUTPUT # set timestamp for cache
-      - name: check if secrets are available
```

Comment:
Why not leave the check-secrets as a step and reference it as I proposed previously?

Reply:
https://github.com/jacktrip/jacktrip/pull/883#discussion_r1084667531